07.ls_object/lib/ls_object.rb

```diff
@@ -5,38 +5,43 @@
 require_relative 'screen'
 
 class LsObject
+  def self.main(paths, options)
+    new(paths, options).main
+  end
+
   def initialize(paths, options)
     @paths = paths.empty? ? ['.'] : paths
     @options = options
```

Comment:
```suggestion
    @options = options
    @options[:header] = @paths.length > 1
```

で最初から全部 `@options` に入れちゃえば？と思いました